README.md

```diff
@@ -156,42 +176,81 @@ If you want to listener the upload results, you can pass in the callback functio
 
 ```js
 await flatDirectory.upload(key, data, {
-    onProgress: function (progress, count) {
+    onProgress: function (progress, count, isChange) {
         console.log(`Uploaded ${progress} of ${totalCount} chunks`);
     },
     onFail: function (err) {
         console.log(err);
     },
-    onSuccess: function (totalUploadChunks, totalUploadSize, totalStorageCost) {
+    onFinish: function (totalUploadChunks, totalUploadSize, totalStorageCost) {
         console.log(`Total upload chunk count is ${totalUploadChunks}, size is ${totalUploadSize}, storage cost is ${totalStorageCost}`);
     }
 });
 ```
 
-#### download
+#### uploadFile
+
+Upload file object to FlatDirectory.
+
+Browser
+```javascript
+// <input id='fileToUpload' />
+const file = document.getElementById('fileToUpload').files[0];
+await flatDirectory.uploadFile(key, file, {
+    onProgress: function (progress, count, isChange) {
+        console.log(`Uploaded ${progress} of ${totalCount} chunks`);
```

Comment:
Replace totalCount with count? and please make similar modifications in other places in this document where the same issue occurs.